pkg/delete/delete_loader_test.go

```diff
@@ -173,6 +173,7 @@ func TestParseDeleteFileDefinitionsWithInvalidDefinition(t *testing.T) {
 }
 
 func TestLoadEntriesToDelete(t *testing.T) {
+	t.Setenv(featureflags.Experimental().EnvName(), "true")
```

Comment:
might have missed it, but could you please add covering the fact that delete filters out disabled APIs? So this can't accidentally break in the future. 
E.g. have a test that sets up a dedicated API map with an experimental one (so it's not impacted by setting current ones to non-experimental), then assert that there no delete entries loaded from a file that defines that API.